lib/plugins/aws/package/lib/validateCompiledTemplate.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const CF = require('aws-sdk/clients/cloudformation');
```

Comment:
To be compliant with the rest of the serverless codebase I would use the request object - for the validateTemplate function it is really not necessary to go through the SDK. The SLS implementation additionally is aware of the stage and region, which is important. If you use the SDK separately you'd have to replicate all the stuff that already is implemented in the core to ensure exactly the same behavior. It even gets worse if the semantics in the SLS request implementation changes for whatever reason.